### PR TITLE
Allure report

### PR DESCRIPTION
**Pull request by AlexeyCL**
Adding functions to allure report and update dependencies

### CODE REVIEW

File: src/test/wdio.conf.js [hunk 1]
```diff
@@ -29,6 +29,20 @@ exports.config = {
    * webdriver service
    */
   services: ['selenium-standalone'],
+  seleniumArgs: {
+    drivers : {
+      chrome : {
+        version : "77.0.3865.40"
```
Comment (FelixZilber) [inline src/test/wdio.conf.js]: this should be variable. travis install latest version

File: src/test/wdio.conf.js [hunk 2]
```diff
@@ -29,6 +29,20 @@ exports.config = {
...
+      }
+    }
+  },
+  seleniumInstallArgs: {
+    drivers : {
+      chrome : {
+        version : "77.0.3865.40"
```
Comment (FelixZilber) [inline src/test/wdio.conf.js]: this should be variable. travis install latest version

File: src/test/wdio.conf.js (hunk 1 above)
Comment (AlexeyCL) [reply src/test/wdio.conf.js]: the code was removed from this PR

File: src/test/wdio.conf.js (hunk 2 above)
Comment (AlexeyCL) [reply src/test/wdio.conf.js]: the code was removed from this PR

**Pull request merged**